.github/workflows/remove-disabled-formulae.yml

```diff
@@ -22,9 +22,16 @@ jobs:
         with:
           username: BrewTestBot
 
+      - name: Set up commit signing
+        uses: Homebrew/actions/setup-commit-signing@master
+        with:
+          signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
+
       - name: Remove disabled formulae
         id: remove_disabled
         uses: Homebrew/actions/remove-disabled-formulae@master
+        env:
```

Comment:
I was unsure if this would work (I'm not too familiar with Actions syntax 😅). If this is incorrect, we could remove the committing step from the action and add it here as a separate step instead.